lib/v2/utils/rate_states_extensions.dart

```diff
@@ -28,4 +31,22 @@ extension RatesStateExtensions on RatesState {
       }
     }
   }
+
+  /// Return a display string or "" if conversion is impossible
+  String fiatValueString(TokenModel token, double? amount) {
```

Comment:
Can we make this take a double and not a nullable double ? 

it will help the code below